--- a/etc/apache-serviceTest.conf
+++ b/etc/apache-serviceTest.conf
@@ -1,37 +1,37 @@
-Alias /serviceTest/JS /opt/perfsonar_ps/serviceTest/JS
-<Directory "/opt/perfsonar_ps/serviceTest/JS">
+Alias /serviceTest/JS /usr/lib/perfsonar/serviceTest/JS
+<Directory "/usr/lib/perfsonar/serviceTest/JS">
     Options FollowSymLinks MultiViews
     AllowOverride None
     Order allow,deny
     Allow from all
 </Directory>
 
-Alias /serviceTest/html /opt/perfsonar_ps/serviceTest/html
-<Directory "/opt/perfsonar_ps/serviceTest/html">
+Alias /serviceTest/html /usr/lib/perfsonar/serviceTest/html
+<Directory "/usr/lib/perfsonar/serviceTest/html">
     Options FollowSymLinks MultiViews
     AllowOverride None
     Order allow,deny
     Allow from all
 </Directory>
 
-Alias /serviceTest/images /opt/perfsonar_ps/serviceTest/images
-<Directory "/opt/perfsonar_ps/serviceTest/images">
+Alias /serviceTest/images /usr/lib/perfsonar/serviceTest/images
+<Directory "/usr/lib/perfsonar/serviceTest/images">
     Options FollowSymLinks MultiViews
     AllowOverride None
     Order allow,deny
     Allow from all
 </Directory>
 
-Alias /serviceTest/css /opt/perfsonar_ps/serviceTest/css
-<Directory "/opt/perfsonar_ps/serviceTest/css">
+Alias /serviceTest/css /usr/lib/perfsonar/serviceTest/css
+<Directory "/usr/lib/perfsonar/serviceTest/css">
     Options FollowSymLinks MultiViews
     AllowOverride None
     Order allow,deny
     Allow from all
 </Directory>
 
-Alias /serviceTest /opt/perfsonar_ps/serviceTest/cgi-bin
-<Directory "/opt/perfsonar_ps/serviceTest/cgi-bin">
+Alias /serviceTest /usr/lib/perfsonar/serviceTest/cgi-bin
+<Directory "/usr/lib/perfsonar/serviceTest/cgi-bin">
     Options FollowSymLinks MultiViews +ExecCGI
     DirectoryIndex index.cgi
     AddHandler cgi-script .cgi
